operator/cmd/mesh/upgrade.go

```diff
@@ -290,17 +297,16 @@ func checkSupportedVersions(kubeClient *Client, currentVersion, targetVersion st
 	if err != nil {
 		return fmt.Errorf("failed to parse the target version %q: %v", targetVersion, err)
 	}
-	if !SupportedIstioVersions.Check(curGoVersion) {
-		return fmt.Errorf("upgrade is currently not supported from version: %v", currentVersion)
+	if upgradeSupportStart.Segments()[1] > curGoVersion.Segments()[1] {
+		return fmt.Errorf("upgrade is not supported before version: %v", upgradeSupportStart)
 	}
 	// Warn if user is trying skip one minor verion eg: 1.6.x to 1.8.x
 	if (targetGoVersion.Segments()[1] - curGoVersion.Segments()[1]) > 1 {
```

Comment:
what about downgrade across more than one minor version?

Reply:
> The Istio version you intend to downgrade to is no more than one minor version less than the installed Istio version. For example, you can downgrade to no lower than 1.6.0 from Istio 1.7.x.

Downgrade prerequisite https://istio.io/latest/docs/setup/upgrade/in-place/#downgrade-prerequisites

Reply:
LGTM